src/EduManagementLab.Api/Controllers/CoursesController.cs

```diff
@@ -133,8 +134,9 @@ public ActionResult DeleteCourse(Guid courseId)
 
 
         [HttpPost()]
+        [Route("{courseId}/Membership")]
         [ProducesResponseType(typeof(CourseDto), StatusCodes.Status200OK)]
-        [Route("{courseId}/Memberships")]
```

Comment:
I think the last merge from master went wrong.
The route should be "{courseId}/Memberships".
See #39